internal/pkg/runtime/engine/singularity/cleanup_host_linux.go

```diff
@@ -20,18 +19,7 @@ import (
 func (e *EngineOperations) CleanupHost(ctx context.Context) (err error) {
 	if e.EngineConfig.GetImageFuse() {
 		sylog.Infof("Unmounting SIF with FUSE...")
-		fusermountPath, err := bin.FindBin("fusermount")
-		if err != nil {
-			return fmt.Errorf("while unmounting fuse directory: %s: %w", e.EngineConfig.GetImage(), err)
-		}
-
-		err = sifuser.Unmount(ctx, e.EngineConfig.GetImage(),
-			sifuser.OptUnmountStdout(os.Stdout),
-			sifuser.OptUnmountStderr(os.Stderr),
-			sifuser.OptUnmountFusermountPath(fusermountPath))
-		if err != nil {
-			return fmt.Errorf("while unmounting fuse directory: %s: %w", e.EngineConfig.GetImage(), err)
-		}
+		squashfs.FUSEUnmount(ctx, e.EngineConfig.GetImage())
```

Comment:
I think we definitely need to handle an error here. If we are unable to unmount in this flow, we will accumulate mounts on the host. The failure should be visible so that the underlying issue has the chance of reaching an admin and being fixed.